server/src/swagger.ts

```
@@ -0,0 +1,253 @@
+const swaggerJSDoc = require('swagger-jsdoc');
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)